main.go

```diff
@@ -118,7 +119,7 @@ func main() {
 
 		artifactURLCollection, err := deploy(deployableItems, config)
 		if err != nil {
-			fail("%s", err)
+			fail(errorutil.FormattedError(err))
```

Comment:
When I checked the linked build in the PR description I have seen this in the build logs:
```
Deploying files

Uploading file: /Users/vagrant/deploy/bigfile
creating artifact
  file size: 3072MB
1 attempt failed
2 attempt failed
3 attempt failed
deploy failed, error:
  failed to create file artifact: /Users/vagrant/deploy/bigfile:
    Artifact files must be 2 GB or smaller, this file is 3 GB.

Uploading file: /Users/vagrant/deploy/hello_world.txt
creating artifact
  file size: 0B
uploading artifact
finishing artifact
deploy failed, error:
  failed to create file artifact: /Users/vagrant/deploy/bigfile:
    Artifact files must be 2 GB or smaller, this file is 3 GB.
```
And first I thought that the `hello_world.txt` upload has failed as well because the last error was together with that upload section. Then I read the error message and realised that this is just the first error printed out again when we return the first error from the `deploy` function.

I would separate this final error printing from the rest for better readability. Something like this:
```
fmt.Println()
fail(errorutil.FormattedError(fmt.Errorf("file deployment failure: %w", err)))
```
and also probably remove `deploy failed, error:` part from the upload errors because with this change that information will be duplicated.

I am not sure that this is the correct wording but could be an option.

Reply:
[4c9bdd0](https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/pull/167/commits/4c9bdd02b51217b0e262ecbb4ad0b7edb4fc091a)

Updated output:
https://app.bitrise.io/build/de169bb5-f556-4506-af60-63704d4cd22d